### DISCUSSION TITLE
Dialog: is there any problem using Alpine.js and TallStackUi in the same project?

### DISCUSSION THREAD

**Discussion by vitorbizarra** [tasteui/tasteui#398] [Q&A] [answered]
Hi! 👋

In a project I have Alpine.js and TallStackUI installed. Everything works normally, but I end up receiving a lot of errors in the console regarding the dialog, such as "Uncaught ReferenceError: dialog is not defined" and "Uncaught ReferenceError: tallstackui_dialog is not defined"

I ended up not finding anything in the documentation. Is there any incompatibility in using TallStackUI and Alpine.js in the same project?

Console:
![image](https://github.com/tallstackui/tallstackui/assets/79993997/fd0f2e2b-21af-4bfb-a84d-4a245accb466)

My `app.blade.php:`
```blade
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <style> [x-cloak] { display: none; } </style> 

    <tallstackui:script />
    @livewireStyles
    @vite(['resources/css/app.css', 'resources/js/app.js'])
    <link rel="shortcut icon" href="{{ asset('favicon.ico') }}" type="image/x-icon">
    <title>{{ isset($title) ? env('APP_NAME') . " - $title" : env('APP_NAME') }}</title>
</head>

<body>
    <x-dialog /> 

    <livewire:web.components.navbar />
    {{ $slot }}
    <livewire:web.components.footer />

    @livewireScripts
</body>

</html>
```

My `app.js:`
```js
import './bootstrap';
import Alpine from 'alpinejs';
import mask from '@alpinejs/mask'
import Swiper from 'swiper/bundle';
import VenoBox from 'venobox';
import 'swiper/css/bundle';
import 'flowbite';

window.Swiper = Swiper;

// CATALOG VENOBOX
new VenoBox({
    selector : ".vb-images",
    fitView : true,
    focusItem : true,
    spinner : "swing",
    titleattr : 'data-title',
    titlePosition : 'bottom',
})

Alpine.plugin(mask);
Alpine.start();
```

**Comment by devajmeireles** [comment, marked as answer]
TallStackUi was created for version 3 of Livewire that delivers AlpineJS by default, along with the mask plugin. You don't have to load AlpineJS more into your app.js. Remove the AlpineJs load and build again. It should solve the problem.

**Reply by vitorbizarra** [reply]
Oh, perfect! I hadn't seen this issue that AlpineJs was already bundled with Livewire 3. I removed it from package.json and all the console errors disappeared like magic hahaha

Thanks!!
